src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -773,12 +775,26 @@ public void updateReferencedBigQueryDataset(
    * @param updateParams resource properties to update
    */
   public void updateControlledBigQueryDataset(
-      UUID workspaceId, UUID resourceId, UpdateResourceParams updateParams) {
+      UUID workspaceId, UUID resourceId, UpdateBqDatasetParams updateParams) {
+
+    GcpBigQueryDatasetUpdateParameters bigQueryDatasetUpdateParameters =
+        new GcpBigQueryDatasetUpdateParameters();
+
+    if (updateParams.partitionExpirationTime.isPresent())
```

Comment:
nit: please use braces around conditionals, at least for the multi-line ones

Reply:
Will do in the future (this statement got taken out in latest commit).